README.md

```diff
@@ -244,6 +244,10 @@ class Client < NxtHttpClient::Client
 end
 ```
 
+NOTE: In order to avoid problems with threaded application servers you should always clear the cache key one your 
+threads once the request is finished using `NxtHttpClient.clear_thread_cache_key` to not have other threads use the 
+same cache key and thus use cached results across threads.
+
```

Comment:
So before a thread could handle a request and then another but keep its cache?

Reply:
Basically if an application server reuses a thread the thread local variable need to be cleared so it's not bleeding into another request. Because the idea of caching per thread actually is to cache within a single request.